packages/dev/core/src/Materials/Node/Blocks/Fragment/prePassOutputBlock.ts

```diff
@@ -89,6 +116,14 @@ export class PrePassOutputBlock extends NodeMaterialBlock {
             state.compilationString += ` gl_FragData[PREPASS_DEPTH_INDEX] = vec4(0.0, 0.0, 0.0, 0.0);\r\n`;
         }
         state.compilationString += `#endif\r\n`;
+        state.compilationString += `#ifdef PREPASS_CLIPSPACE_DEPTH\r\n`;
+        if (viewDepthClipSpace.connectedPoint) {
+            state.compilationString += ` gl_FragData[PREPASS_CLIPSPACE_DEPTH_INDEX] = vec4(${viewDepthClipSpace.associatedVariableName}, 0.0, 0.0, 1.0);\r\n`;

```

Comment:
This happens after you did that PR but now this file needs update to also support WebGPU (the changes should be straight forward)

Reply:
Done